src/net_processing.cpp

```diff
@@ -452,16 +453,26 @@ class PeerManagerImpl final : public PeerManager
      *
      * Memory used: 1.3 MB
      */
-    std::unique_ptr<CRollingBloomFilter> recentRejects GUARDED_BY(cs_main);
+    CRollingBloomFilter m_recent_rejects GUARDED_BY(cs_main) {120000, 0.000001};
```

Comment:
6e48bfe706934fa4b71a3021d979321e94343c21, style nit:
```suggestion
    CRollingBloomFilter m_recent_rejects GUARDED_BY(::cs_main){120'000, 0.000001};
```

([`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) states that the space is redundant)

Reply:
Done. Thanks!